src/lib/instance_metadata.py

```diff
@@ -93,7 +93,7 @@ async def execute(self) -> Optional[InstanceMetadata]:
             results = await asyncio.gather(*metadata, return_exceptions=True)
 
             if not all(result is None for result in results):
-                audience = jwt.decode(results[4], verify=False) if results[4] else ''
+                audience = jwt.decode(results[4], options={"verify_signature": False}) if results[4] else ''
```

Comment:
Ref: https://pyjwt.readthedocs.io/en/2.4.0/changelog.html#dropped-deprecated-verify-param-in-jwt-decode